pom.xml

```diff
@@ -98,6 +98,7 @@
                     <autoReleaseAfterClose>true</autoReleaseAfterClose>
                 </configuration>
             </plugin>
+            <!-- uncomment when uploading to maven central
```

Comment:
This makes `mvn verify` fail if you don't have gpg set up, but it is only required when releasing.